Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RazorTargetTests.cs

```diff
@@ -37,29 +38,79 @@ public class RazorTargetTests
 
         public TestContext TestContext { get; set; }
 
+        [TestMethod]
+        public void Razor_RazorSpecificOutputAndProjectInfo_AreCopiedToCorrectFolders()
+        {
+            // Arrange
+            var root = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
+            var projectSpecificOutDir = Path.Combine(root, "0");
+            var temporaryProjectSpecificOutDir = Path.Combine(root, "0.tmp");
+            var razorSpecificOutDir = Path.Combine(root, "0.Razor");
+            Directory.CreateDirectory(projectSpecificOutDir);
+            Directory.CreateDirectory(temporaryProjectSpecificOutDir);
+            File.WriteAllText(Path.Combine(projectSpecificOutDir, "ProjectInfo.xml"),
+$@"
```

Comment:
This whole XML looks messed up by some bad copy pasting - there are many spaces on places where they should not be
```
<Platform > AnyCPU </ Platform >
<Platform>AnyCPU</Platform>
```